src/lib/seam/devicedb/public-models/manufacturer.ts

```diff
@@ -21,6 +21,7 @@ export const manufacturer = z.object({
   integration: manufacturer_integration_support_level,
   is_connect_webview_supported: z.boolean(),
   requires_seam_support_to_add_account: z.boolean(),
+  device_model_count: z.number().optional(),
```

Comment:
@codetheweb Since we have this above,

<img width="315" alt="image" src="https://github.com/seamapi/types/assets/721372/8faf54a5-9c23-4977-9d72-63a68c650602">

could we set this to non-optional now?

Reply:
I'm not sure the types will work out without some refactoring but feel free to try